### PR TITLE
CME-247: Navigation component

### PR DESCRIPTION
**Pull request by bmonedeiro**

### CODE REVIEW

File: components/magnolia/components/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  BreadcrumbItem,
+  BreadcrumbLink,
+  Button,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+} from '@chakra-ui/react';
+import { languages } from '~/utils';
```
Comment (wmoffett) [inline components/magnolia/components/Navigation.tsx]: languages exist in @components/magnolia/api, use getLanguages() lets keep every thing together until we figure out what our core library might look like. 
I'm not a big fan of util files LOL ;)

File: utils.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const languages =
```
Comment (wmoffett) [inline utils.ts]: This all exists in @components/magnolia/api no need to add this file or the methods.

**Review by wmoffett — CHANGES_REQUESTED** [review]
Please see my comments.

File: components/magnolia/components/Navigation.tsx (hunk 1 above)
Comment (bmonedeiro) [reply components/magnolia/components/Navigation.tsx]: done! 😊

File: utils.ts (hunk 2 above)
Comment (bmonedeiro) [reply utils.ts]: sure! 😊

**Review by wmoffett — APPROVED** [review]
LGTM. we may want to add the nave to the static generated pages as well.

**Pull request merged**